services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet[Movie]:
+
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        return Movie.objects.all().filter(
+            genres__id__in=genres_ids, actors__id__in=actors_ids
+        )
+
+    if genres_ids and not actors_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+
+    if actors_ids and not genres_ids:
+        return Movie.objects.filter(actors__id__in=actors_ids)
```

Comment:
Leave only one return statement in this method. You could define an initial `queryset` variable at the beginning, add two if conditions to add filtration to queryset if corresponding arguments are present. Remember that querysets are lazy